src/actions/users-me.js

```diff
@@ -140,12 +140,8 @@ export const changePassword = createHttpAction(
     password,
     new_password,
     new_password_confirmation,
-    { dispatch, getState }
   ) => {
-    const { me } = getState();
-    return dispatch(authenticateFresh(me.username, password)).then((r) =>
-      backend.changePassword(password, new_password, new_password_confirmation)
-    );
+     return backend.changePassword(password, new_password, new_password_confirmation)
```

Comment:
`new_password_confirmation` is not validated on backend now, is it validated somewhere on FE?

Reply:
yes, the form validates it and shows error message if there is a mismatch. no request to backend is sent